### ISSUE TITLE
NullPointerException in server side during closing stream

### ISSUE THREAD

**Issue by tukez** [salesforce/reactive-grpc#221]
I got a bug report for a NPE, here is the stack from the report:

```
INFO 2020-06-24 10:36:50,653 [BackendScheduler-11] AuditLog - <redacted>
kesäkuuta 24, 2020 10:37:02 AP. io.grpc.internal.SerializingExecutor run
SEVERE: Exception while executing runnable io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1Closed@66bcc3a
java.lang.NullPointerException
at com.salesforce.reactivegrpc.common.AbstractSubscriberAndProducer.cancel(AbstractSubscriberAndProducer.java:113)
at com.salesforce.reactivegrpc.common.AbstractSubscriberAndServerProducer.access$001(AbstractSubscriberAndServerProducer.java:18)
at com.salesforce.reactivegrpc.common.AbstractSubscriberAndServerProducer$1.run(AbstractSubscriberAndServerProducer.java:27)
at io.grpc.stub.ServerCalls$StreamingServerCallHandler$StreamingServerCallListener.onCancel(ServerCalls.java:269)
```

Thats all I have on this currently, sorry. I couldn't reproduce this on my local machine.

Looking at the `AbstractSubscriberAndProcuder.cancel`, it seems to me like it's just missing a null check. I can't say is calling the cancel-method in that situation valid though.

**Comment by rmichela** [comment]
Thanks for the report!

**Comment by rmichela** [comment]
Released in 1.1.0
